world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+")
+(quest:interaction :name sign-main-check :interactable sign-main :dialogue "
+~ player
+| (It's the same symbol that I have on my uniform.)
+| (Perhaps it's a warning...) 
+| ! eval (activate 'fi-sign-main)
+| ! eval (complete 'fi-check)")
+(quest:interaction :name fi-sign-main :interactable fi :dialogue "
+~ player
+| That sign over there has the same symbol on it as my uniform. I think it has something to do with androids.
+~ fi
+| Then we're on the right track.
+~ player
+| It also resembles an arrow, don't you think? Pointing up over the mountain?
+~ fi
+| I know you want to be a detective, but now I think you're seeing things.
+| Remind me to check your pattern recognition software when we get back.
+| [has-more-dialogue You got more? | You'd better get going.]")
+(quest:interaction :name sign-check :interactable sign :dialogue "
+~ player
+| (The words were scratched into the wood with a knife. I can barely read it. Upscaling resolution and enabling zoom...)
+| //READ ME - NO LITTER U FUCKS - BOTS PATROL HERE//
+| (Short and not so sweet - that's Harris.)
+| ! eval (activate 'fi-sign)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-sign :interactable fi :dialogue "
+~ player
+| There's a sign in the cave - looked and sounded like only Harris can. It warned of robots on patrol.
+~ fi
+| Rogues... They're not uncommon in caves like these. They're like you but not as strong - and thankfully not as smart.
+~ player
+| Was that a compliment?
+~ fi
+| A fact.
+| [has-more-dialogue Anything else? | Now get going.]")
+(quest:interaction :name landslide-check :interactable landslide :dialogue "
+~ player
+| (A landslide... That's a memory I could do without.)
+| (There's human blood on the rock. I can't follow them through there - I need to find another way around.)
+| ! eval (activate 'fi-landslide)
+| ! eval (complete 'fi-check)
+")
+; | ! eval (let (landslide true))
+(quest:interaction :name fi-landslide :interactable fi :dialogue "
+~ player
+| There's been a landslide in the cave.
+~ fi
+| That's not unusual in these parts. But given that everything else seems wrong about this place...
+~ player
+| I hope the android isn't buried under there.
+~ fi
+| Androids are tougher than they look.
+~ player
+| I don't look tough?
+~ fi
+| *Smiles* I didn't say that.
+| [has-more-dialogue Anything more? | Well this is awkward...]")
+(quest:interaction :name fire-check :interactable fire :dialogue "
+~ player
+| (The fire is still warm. They left recently.)
+| ! eval (activate 'fi-fire)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-fire :interactable fi :dialogue "
+~ player
+| I found the remnants of another human camp. Recently vacated.
+~ fi
+| I think it's fair to say that Harris hasn't been completely honest with us.
+~ player
+| You think it's a trap?
+~ fi
+| There's one way to find out.
+~ player
+| Yes... Lucky me.
+~ fi
+| You can handle yourself.
+| [has-more-dialogue Next? | Sayonara.]")
```

Comment:
heh, nice, yeah I do like a bit of language code-switching from time to time, to show more accent and dialect. Stylistically for consistency I think it would still be written in English here